strawberry/test/client.py

```diff
@@ -149,7 +149,10 @@ def _build_multipart_file_map(
             else:
                 map[key] = [f"variables.{reference}"]
 
-        return map
+        # remove the other variables that are not files, we can't remove them before
+        # because they can be part of a list of files or folder
+        map_without_vars = {k: v for k, v in map.items() if k in files.keys()}
+        return map_without_vars
```

Comment:
can you explain why we remove them in the comment? 😊

Reply:
sure!